udata_front/theme/gouvfr/assets/js/config.ts

```diff
@@ -288,7 +288,7 @@ export const search_autocomplete_debounce = _jsonMeta("search-autocomplete-debou
 
 export const tabular_api_url = _meta("tabular-api-url");
 
-export const tabular_page_size = _meta("tabular-page-size");
+export const tabular_page_size = parseInt(_meta("tabular-page-size") ?? "10");
```

Comment:
Why can't it be left to null if not defined?

Reply:
It is always defined in `udata_tabular_preview`, but Typescript doesn't know it and we can't parse a `string | null` to `int`. I will changed the default to match the one from `udata_tabular_preview` and datagouv/components.